docs/features/collision/collision.md

```diff
@@ -25,7 +151,10 @@ public interface IEntity : ICollisionActor
 
 Next, we define our entity classes
 
-The `OnCollision` method and the Bounds property come from the `ICollisionActor` interface. These will be called by the `CollisionComponent`
+The `OnCollision` method and the Bounds property come from the `ICollisionActor` interface. These will be called by the `CollisionComponent`.
+:::note[notice]
+The RandomizeVector method will give a warning/error, this will be resolved in the "Setting up the game" section when you add the Random variable to the Game1.cs file.
```

Comment:
Unless there is a specific reason someone needs to seed the `Random`, you can switch this from using `_game.Random` to instead use `Random.Shared` which will no longer need this mention of a warning/error.